app/assets/v2/js/hackathon-projects.js

```diff
@@ -68,6 +72,15 @@ const projectModal = (bountyId, projectId, callback) => {
     });
   });
 
+  $('#videodemo-url').keyup(function(event) {
```

Comment:
@zoek1 it seems this block of code is unreached cause the video url isn't saved when i submit.

Reply:
i get `Error in render: "TypeError: Cannot read property 'url' of undefined` in the console when i submit.

Reply:
There's two things here: the first is keyup listen on changes on the field with the id #videodemo-url, this doesn't requires an existing value, it will evaluate while the user write something on such field. 
The second Is a good catch, i fixed that with the latest commit also that prevent refresh the data in the page. Thanks for review It!

Reply:
@zoek1 

it second issue has been resolved. 👍 💯  

for the first issue about video not saving in db, kindly observe the recording below,

https://www.loom.com/share/406b9513467c4fb7967362ba152b9c4b

if `keyup` doesn't work as expected, don't you think you should `getVideoMetadata` on submit instead?

ps: i tested on google chrome browser.

Reply:
^ @PixelantDesign / @zoek1 will have to wait until this gets resolved